packages/react-core/src/modules/in-app-message/hooks/useInAppMessages.ts

```diff
@@ -12,13 +12,19 @@ import { useInAppChannel } from "../context";
 
 export interface UseInAppMessagesOptions extends InAppMessagesClientOptions {}
 
-export interface UseInAppMessagesResponse<TContent, TData> {
+export interface UseInAppMessagesResponse<
```

Comment:
Not for this PR, but again later we should take a full pass at removing all exports that aren't used outside and meant to be used by end users.

Reply:
Ack, got a todo for that